rpcs3/Emu/Cell/Modules/cellGcmSys.cpp

```diff
@@ -303,7 +303,24 @@ error_code cellGcmBindZcull(u8 index, u32 offset, u32 width, u32 height, u32 cul
 		return CELL_GCM_ERROR_INVALID_VALUE;
 	}
 
-	rsx::get_current_renderer()->zculls[index].bound = true;
+	auto& gcm_cfg = g_fxo->get<gcm_config>();
```

Comment:
Out of curiosity - is it feasible to pipe these kinds of GCM functions to sys_rsx_context_attribute? It would allow us to consolidate the logic into one file. I'm not saying it needs to be done right away of course.